README.md

```diff
@@ -285,6 +285,7 @@ https://github.com/sdras/awesome-actions/workflows/Lint%20Awesome%20List/badge.s
 - [Automatically Update PRs with Outdated Checks and Squash and Merge the Ones Matching All Branch Protections](https://github.com/tibdex/autosquash)
 - [Merge Pal - automatically update and merge pull requests](https://github.com/maxkomarychev/merge-pal-action)
 - [Enforce naming convention on pull request title](https://github.com/deepakputhraya/action-pr-title)
+- [Pull Request Stuck Notifier](https://github.com/marketplace/actions/pull-request-stuck-notifier)
```

Comment:
@jayrylan is there a reason that you are linking to the marketplace for this action?  All links in this repository point to the GitHub Repository behind the action.

Reply:
@gep13 Sorry about that, I've corrected the link and rebased from master. Thank you!